components/PageNavigation.vue

```diff
@@ -101,9 +154,83 @@
       }
     }
     @media (max-width: $bp-large) {
-      .nav-link.nuxt-link-active {
-        &:after {
-          bottom: -1rem;
+      width: 100%;
+      margin: 0 0 0.25rem 0;
+      position: relative;
+      margin-right: 0;
+      &:nth-last-child(2) {
+        margin-right: 0;
+      }
+      &:first-of-type, &:last-of-type {
+        display: block;
+      }
+      .nav-link {
+        text-transform: capitalize;
+        font-weight: 400;
+        border-radius: $border-radius-small;
+        transition: $standard-transition;
+        font-size: $font-size-base;
+        span {
+          display: flex;
+          align-items: center;
+          i {
+            display: inline-block;
+            font-size: 1rem;
+            z-index: 1;
+            margin-right: 0.75rem;
+            &:before {
+              content: '';
+              color: $black;
+              transition: $standard-transition;
+              font-size: 1.5rem;
+            }
+            &.icon-home:before {
+              content: '\e922';
+            }
+            &.icon-collections:before {
+              content: '\e91d';
+            }
+            &.icon-school:before {
+              content: '\e91e';
+            }
+            &.icon-info:before {
+              content: '\e91f';
+            }
+            &.icon-help:before {
+              content: '\e921';
+            }
```

Comment:
It looks like there is some duplication of code here, am I correct? The `span` and `i` and icons are also on line 113-140.

Reply:
sorry, have removed now